src/MultipartStreamBuilder.php

```diff
@@ -140,9 +140,7 @@ public function build()
             fwrite($buffer, "--{$this->getBoundary()}\r\n".
                 $this->getHeaders($data['headers'])."\r\n");
 
-            /**
-             * @var \Psr\Http\Message\StreamInterface
-             */
+            /* @var $contentStream StreamInterface */
```

Comment:
Needs to have two stars, not one. A single star is a comment, and not phpdoc. If php-cs-fixer is messing this up, you probably need to replace this with `/** @var StreamInterface */` and/or just disable the broken fixer.